server/boot.mjs

```diff
@@ -0,0 +1,26 @@
+import {build} from 'vite';
+import path from 'path';
+import {execSync} from 'child_process';
+
+process.env.NODE_ENV='production';
+
+(async () => {
+  /** Create Build */
+
+  await build({
+    root: '.',
+    base: './',
+  });
+
+  console.log('Copying Scenes');
+  /** Copy scene directory */
+  execSync('cp -r ./scenes ./dist/scenes');
+
+  console.log('Copying Metaverse_Modules');
+  /** Copy Metaverse directory */
+  execSync('cp -r ./metaverse_modules ./dist/metaverse_modules');
```

Comment:
What if FS is not writable?

Also executing shell is not the best idea. Will this even work on windows?